ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -479,37 +483,58 @@ def test_frontend_function(
         for v, d in zip(as_variable_flags, input_dtypes)
     ]
 
-    # parse function name and frontend submodules (jax.lax, jax.numpy etc.)
-    *frontend_submods, fn_name = fn_tree.split(".")
-    function_dict = ivy.functional.frontends.__dict__[frontend]
+    def _get_function(fn_tree):
+        # parse function name and frontend submodules (jax.lax, jax.numpy etc.)
+        *frontend_submods, fn_name = fn_tree.split(".")
+        function_dict = ivy.functional.frontends.__dict__[frontend]
+
+        # Getting function attributes when we have function tree such as
+        # nn.functional etc
+        len_frontend_submods = len(frontend_submods)
+        if len_frontend_submods > 0:
```

Comment:
We can simplify this by using `importlib.import_module`, we don't have to manually trace it.

Reply:
Yeah sure we can simplify it and do proper concatenations for frontend modules strings